src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -95,7 +92,6 @@ const defaultProps = {
     currentUserPersonalDetails: {
         avatar: '',
     },
-    reportIDFromRoute: '',
```

Comment:
Add `currentReportId`

Reply:
`currentReportId` is included in `withCurrentReportIdPropTypes`, no need add `currentReportId`.

Reply:
Added.

Reply:
Adding `currentReportId` fails on the Lint test, since no propTypes declaration in `SideBarLinks.js`.
But I think adding propTypes is redundant.
`currentReportId` is included in `withCurrentReportIdPropTypes`, so no need to be added.
Could you please give me know your opinion on this?

Reply:
yeah let's revert last commit

Reply:
Reverted.